ee/localserver/request-id.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/kolide/kit/ulid"
 	"github.com/kolide/launcher/ee/consoleuser"
 	"github.com/kolide/launcher/pkg/backoff"
+	"github.com/kolide/launcher/pkg/traces"
+	"go.opentelemetry.io/otel/codes"
```

Comment:
I don't love needing codes here. But maybe it's a good enough

Reply:
Updated traces pkg to add a `SetError` function that handles this, so we don't need codes here anymore